deckhand/engine/schema/v1_0/default_schema.py

```diff
@@ -28,16 +28,12 @@
         'src': {
             'type': 'object',
             'properties': {
-                'apiVersion': {
```

Comment:
I think we can keep this basic form validation, but you're right that we should validate against known schema names as well (later once that's in place).